<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Televideo</class>
 <widget class="QDialog" name="Televideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>635</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TeleDrake v.0.1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recoverdrake.qrc">
    <normaloff>:/Imagenes/Television.png</normaloff>:/Imagenes/Television.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="Phonon::VideoPlayer" name="videoPlayer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>220</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="8">
          <widget class="Phonon::VolumeSlider" name="volumeSlider_6" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>251</width>
             <height>21</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Play</string>
           </property>
           <property name="icon">
            <iconset resource="recoverdrake.qrc">
             <normaloff>:/Imagenes/play.png</normaloff>:/Imagenes/play.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Pausa</string>
           </property>
           <property name="icon">
            <iconset resource="recoverdrake.qrc">
             <normaloff>:/Imagenes/pause.png</normaloff>:/Imagenes/pause.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Abrir</string>
           </property>
           <property name="icon">
            <iconset resource="recoverdrake.qrc">
             <normaloff>:/Imagenes/abrir.png</normaloff>:/Imagenes/abrir.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="pushButton_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Salir</string>
           </property>
           <property name="icon">
            <iconset resource="recoverdrake.qrc">
             <normaloff>:/Imagenes/puerta.png</normaloff>:/Imagenes/puerta.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="pushButton_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Pantalla completa</string>
           </property>
           <property name="icon">
            <iconset resource="recoverdrake.qrc">
             <normaloff>:/Imagenes/tv.png</normaloff>:/Imagenes/tv.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="icon">
            <iconset resource="recoverdrake.qrc">
             <normaloff>:/Imagenes/stop1.png</normaloff>:/Imagenes/stop1.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_9">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Cargar sop</string>
             </property>
             <property name="icon">
              <iconset resource="recoverdrake.qrc">
               <normaloff>:/Imagenes/Television.png</normaloff>:/Imagenes/Television.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>Quitar emisora</string>
             </property>
             <property name="icon">
              <iconset resource="recoverdrake.qrc">
               <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>Ocultar lista</string>
             </property>
             <property name="icon">
              <iconset resource="recoverdrake.qrc">
               <normaloff>:/Imagenes/sustituir.png</normaloff>:/Imagenes/sustituir.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recoverdrake.qrc"/>
 </resources>
 <connections/>
</ui>
